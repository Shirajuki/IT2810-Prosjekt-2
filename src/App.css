.App {
  text-align: center;
}
.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  #peace {
    animation: App-logo-spin 10s linear infinite;
    transform-origin: 200px 200px;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 10vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
  margin-bottom: 20px;
}

.App-link {
  color: #61dafb;
}
.modal-container {
  position: fixed; /* Stay in place */
  z-index: 1; /* Sit on top */
  left: 0;
  top: 0;
  width: 100%; /* Full width */
  height: 100%; /* Full height */
  overflow: auto; /* Enable scroll if needed */
  background-color: rgb(0, 0, 0); /* Fallback color */
  background-color: rgba(0, 0, 0, 0.7); /* Black w/ opacity */
}

#modal-content {
  background-color: #161616;
  /* margin: 15% auto; /* 15% from the top and centered */
  padding: 20px;
  border: 1px solid #888;
  width: 80%; /* Could be more or less, depending on screen size */
  position: absolute;
  left: 50%;
  top: 50%;
  transform: translate(-50%, calc(-50% + 25px));
  max-height: calc(100vh - 150px);
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
