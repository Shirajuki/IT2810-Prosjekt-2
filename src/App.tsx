import React, { useState } from "react";
import "./App.css";
import Installation from "./Installation";
import { IoMdClose } from "react-icons/io";
import Poem from "./Poem";
import Slideshow from "./Slideshow";
import Test from "./SVG_installations/SVGtest";
import Moon from "./canvas_installations/moon";
import Snow from "./canvas_installations/snow";

import BlackForest from "./SVG_installations/BlackForest";
import Birch from "./canvas_installations/birch";
import WallFlower from "./canvas_installations/wallFlower";
var background = "";
var header = "";
var hover = "";

function setTheme(theme = "") {
  if (theme === "black") {
    background = "#171717";
    header = "rgba(23, 23, 23, 0.9)";
    hover = "#101010";
  } else if (theme === "white") {
    background = "#FFF";
    header = "rgba(200, 200, 200, 0.9)";
    hover = "#101010";
  } else if (theme === "pink") {
    background = "pink";
    header = "darkpink";
    hover = "#101010";
  }
  document.documentElement.style.setProperty("--background-colors", background);
  document.documentElement.style.setProperty("--header-color", header);
  document.documentElement.style.setProperty("--hover-color", hover);
}

function App() {
  const [modal, setModal] = useState({
    title: "",

    sound: "",
  });
  return (
    <>
      <div className="wrapper">
        <header>
          <div>
            <h1>
              <a href="index.html" className="logo">
                Kunst ustilling
              </a>
            </h1>
            <nav>
              <a onClick={() => setTheme("pink")}>Pink</a>
              <a onClick={() => setTheme("white")}>Light</a>
              <a onClick={() => setTheme("black")}>Dark</a>
            </nav>
          </div>
        </header>
        <br />
        <main>
          <Slideshow />
          <div className="sectionsplit"></div>
          <div className="sectionsplit"></div>
          <div className="tabBox">
            <a href="#">Tab 1</a>
            <a href="#">Tab 2</a>
          </div>
          <div className="artWrapper">
            <div className="art1">
              <div>
                <BlackForest
                  onClick={() =>
                    setModal({
                      title: "In the Black Forest",
                      sound: "audiofiles/Kalimba.mp3",
                    })
                  }
                ></BlackForest>
              </div>
              <div>
                <div className="poetryBox">
                  <h2 className="titlefont">In The Black Forest</h2>
                  <Poem title={"In the Black Forest"}></Poem>
                </div>
              </div>
            </div>
            <div className="art2">
              <div>
                <Birch
                  onClick={() =>
                    setModal({
                      title: "The Birch-Tree at Loschwitz",

                      sound: "audiofiles/Kalimba.mp3",
                    })
                  }
                ></Birch>
              </div>
              <div>
                <div className="poetryBox reversed">
                  <h2 className="titlefont">The Birch-Tree at Loschwitz</h2>
                  <Poem title={"The Birch-Tree at Loschwitz"}></Poem>
                </div>
              </div>
            </div>
            <div className="art3">
              <div>
                <WallFlower
                  onClick={() =>
                    setModal({
                      title: "A Wall Flower",
                      sound: "audiofiles/Kalimba.mp3",
                    })
                  }
                ></WallFlower>
              </div>
              <div>
                <div className="poetryBox">
                  <h2 className="titlefont">A Wall Flower</h2>
                  <Poem title={"A Wall Flower"}></Poem>
                </div>
              </div>
            </div>
            <div className="art2">
              <div>
                <Snow
                  onClick={() =>
                    setModal({
                      title: "Winter Song",
                      sound: "audiofiles/Kalimba.mp3",
                    })
                  }
                ></Snow>
              </div>
              <div>
                <div className="poetryBox reversed">
                  <h2 className="titlefont">Winter Song</h2>
                  <Poem title={"Winter Song"}></Poem>
                </div>
              </div>
            </div>
          </div>
          <div className="sectionsplit"></div>
          <div id="root"></div>
        </main>
      </div>
      <div className="sectionsplit2"></div>
      <div
        className="modal-container"
        style={modal.title !== "" ? { display: "block" } : { display: "none" }}
      >
        <div id="modal-content">
          <div
            className="close-button"
            id="myBtn"
            onClick={() => setModal({ title: "", sound: "" })}
          >
            <IoMdClose />
          </div>

          <Installation title={modal.title} sound={modal.sound}></Installation>
        </div>
      </div>

      <footer>
        <p>this is a footer</p>
      </footer>
    </>
  );
}
export default App;
