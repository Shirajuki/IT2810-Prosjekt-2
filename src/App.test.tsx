import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import TestRenderer from 'react-test-renderer';
//import 'jest-canvas-mock';
// https://testing-library.com/docs/react-testing-library/api
// https://itnext.io/testing-components-with-jest-and-react-testing-library-d36f5262cde2

describe('Testing for component: <App />', () => {
	it('renders the component correctly without crashing', () => {
		const testRenderer = TestRenderer.create(<App />).toJSON();
		expect(testRenderer).toMatchSnapshot();
	});
});


