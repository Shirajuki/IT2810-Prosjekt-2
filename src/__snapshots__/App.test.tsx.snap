// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> spec renders the component 1`] = `
<div
  class="wrapper"
>
  <header>
    <div>
      <h1>
        <a
          class="logo"
          href="index.html"
        >
          Kunst utstilling
        </a>
      </h1>
      <nav>
        <a
          href="#"
        >
          nav1
        </a>
        <a
          href="#"
        >
          nav2
        </a>
        <a
          href="#"
        >
          nav3
        </a>
        <a
          href="#"
        >
          nav4
        </a>
      </nav>
    </div>
  </header>
  <br />
  <main>
    <div
      class="Slideshow"
    >
      <div
        class="splash"
      >
        <div
          class="div1 active"
        >
          <img
            alt=""
            src="https://via.placeholder.com/750x450?text=1"
          />
          <div>
            <p
              class="titlefont"
            >
              Test1
            </p>
          </div>
        </div>
        <div
          class="div2 inactive"
        >
          <img
            alt=""
            src="https://via.placeholder.com/750x450?text=2"
          />
          <div>
            <p
              class="titlefont"
            >
              Test2
            </p>
          </div>
        </div>
        <div
          class="div3 inactive"
        >
          <img
            alt=""
            src="https://via.placeholder.com/750x450?text=3"
          />
          <div>
            <p
              class="titlefont"
            >
              Test3
            </p>
          </div>
        </div>
        <div
          class="div4"
        >
          <div>
            <h1
              class="title"
            >
              Test1
            </h1>
          </div>
        </div>
        <div
          class="div5"
        >
          <img
            alt=""
            src="https://via.placeholder.com/750x450?text=1"
          />
          <div
            class="btnWrapper"
          >
            <a
              class="button"
            >
              &gt;&gt;
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="bigPoetry"
    >
      <div
        class="poetryBox"
      >
        <p>
          Red lights green lights strawberry lime
        </p>
        <p>
          A girl just died and she was a friend of mine
        </p>
        <p>
          We used to sing and dance and twirl in the pines
        </p>
        <p>
          But now shes gone and she was a friend of mine
        </p>
        <p>
          laugh out loud and cry inside
        </p>
        <p>
          life’ so large... Who’s in charge
        </p>
        <p>
          lost track of time
        </p>
        <p>
          Now she’s gone. And she was a friend of mine
        </p>
        <p>
          I never thought I’d feel the way I do now
        </p>
        <p>
          I never thought I could feel so much pain
        </p>
        <p>
          Oh God please. Wait...God can’t stop the rain
        </p>
        <p>
          Now she’s gone and she was a friend of mine.
        </p>
      </div>
    </div>
    <div
      class="sectionsplit"
    />
    <div
      class="sectionsplit"
    />
    <div
      class="tabBox"
    >
      <a
        href="#"
      >
        Tab 1
      </a>
      <a
        href="#"
      >
        Tab 2
      </a>
    </div>
    <div
      class="artWrapper"
    >
      <div
        class="art1"
      >
        <div>
          <img
            src="a.jpg"
          />
        </div>
        <div>
          <div
            class="poetryBox"
          >
            <h2
              class="titlefont"
            >
              Art title here
            </h2>
            <p>
              Red lights green lights strawberry lime
            </p>
            <p>
              A girl just died and she was a friend of mine
            </p>
            <p>
              We used to sing and dance and twirl in the pines
            </p>
            <p>
              But now shes gone and she was a friend of mine
            </p>
            <p>
              laugh out loud and cry inside
            </p>
            <p>
              life’ so large... Who’s in charge
            </p>
            <p>
              lost track of time
            </p>
            <p>
              Now she’s gone. And she was a friend of mine
            </p>
            <p>
              I never thought I’d feel the way I do now
            </p>
            <p>
              I never thought I could feel so much pain
            </p>
            <p>
              Oh God please. Wait...God can’t stop the rain
            </p>
            <p>
              Now she’s gone and she was a friend of mine.
            </p>
          </div>
        </div>
      </div>
      <div
        class="art2"
      >
        <div>
          <img
            src="a.jpg"
          />
        </div>
        <div>
          <div
            class="poetryBox reversed"
          >
            <h2
              class="titlefont"
            >
              Art title here
            </h2>
            <p>
              Red lights green lights strawberry lime
            </p>
            <p>
              A girl just died and she was a friend of mine
            </p>
            <p>
              We used to sing and dance and twirl in the pines
            </p>
            <p>
              But now shes gone and she was a friend of mine
            </p>
            <p>
              laugh out loud and cry inside
            </p>
            <p>
              life’ so large... Who’s in charge
            </p>
            <p>
              lost track of time
            </p>
            <p>
              Now she’s gone. And she was a friend of mine
            </p>
            <p>
              I never thought I’d feel the way I do now
            </p>
            <p>
              I never thought I could feel so much pain
            </p>
            <p>
              Oh God please. Wait...God can’t stop the rain
            </p>
            <p>
              Now she’s gone and she was a friend of mine.
            </p>
          </div>
        </div>
      </div>
      <div
        class="art3"
      >
        <div>
          <img
            src="a.jpg"
          />
        </div>
        <div>
          <div
            class="poetryBox"
          >
            <h2
              class="titlefont"
            >
              Art title here
            </h2>
            <p>
              Red lights green lights strawberry lime
            </p>
            <p>
              A girl just died and she was a friend of mine
            </p>
            <p>
              We used to sing and dance and twirl in the pines
            </p>
            <p>
              But now shes gone and she was a friend of mine
            </p>
            <p>
              laugh out loud and cry inside
            </p>
            <p>
              life’ so large... Who’s in charge
            </p>
            <p>
              lost track of time
            </p>
            <p>
              Now she’s gone. And she was a friend of mine
            </p>
            <p>
              I never thought I’d feel the way I do now
            </p>
            <p>
              I never thought I could feel so much pain
            </p>
            <p>
              Oh God please. Wait...God can’t stop the rain
            </p>
            <p>
              Now she’s gone and she was a friend of mine.
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sectionsplit"
    />
    <div
      id="root"
    />
  </main>
</div>
`;

exports[`Mulig å få delt opp som dette også <App /> spec renders the component 1`] = `
<div
  class="wrapper"
>
  <header>
    <div>
      <h1>
        <a
          class="logo"
          href="index.html"
        >
          Kunst ustilling
        </a>
      </h1>
      <nav>
        <a
          href="#"
        >
          nav1
        </a>
        <a
          href="#"
        >
          nav2
        </a>
        <a
          href="#"
        >
          nav3
        </a>
        <a
          href="#"
        >
          nav4
        </a>
      </nav>
    </div>
  </header>
  <br />
  <main>
    <div
      class="Slideshow"
    >
      <div
        class="splash"
      >
        <div
          class="div1 active"
        >
          <canvas
            height="450"
            width="750"
          />
          <div>
            <p
              class="titlefont"
            >
              A Wall Flower
            </p>
          </div>
        </div>
        <div
          class="div2 inactive"
        >
          <div
            class="canvas"
          >
            <svg
              viewBox="0 0 750 450"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs>
                <pattern
                  height="40"
                  id="grass"
                  patternUnits="objectBoundingBox"
                  width="60"
                  x="0"
                  y="0"
                >
                  <path
                    d="m 25 38 l -20 -20 l 15 5 z"
                    fill="cadetblue"
                  />
                  <path
                    d="m 25 38 l 35 -30 l -25 10 z"
                    fill="#608da0"
                  />
                </pattern>
                <pattern
                  height="60"
                  id="tree"
                  patternUnits="userSpaceOnUse"
                  width="55"
                  x="-10"
                  y="-10"
                >
                  <rect
                    fill="black"
                    height="30"
                    width="13"
                    x="20"
                    y="30"
                  />
                  <path
                    d="m 0 52 l 27 -35 l 30 35 z"
                    fill="#black"
                  />
                  <path
                    d="m 2 42 l 25 -35 l 26 35 z"
                    fill="#black"
                  />
                  <path
                    d="m 4 32 l 23 -30 l 24 30 z"
                    fill="#black"
                  />
                </pattern>
                <pattern
                  height="60"
                  id="cross"
                  patternUnits="objectBoundingBox"
                  width="40"
                  x="0"
                  y="0"
                >
                  <rect
                    fill="white"
                    height="50"
                    width="40"
                    x="0"
                    y="0"
                  />
                </pattern>
              </defs>
              <g>
                <rect
                  fill="#0f0f18"
                  height="450"
                  width="750"
                  x="0"
                  y="0"
                />
                <circle
                  cx="34"
                  cy="46"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="131"
                  cy="53"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="105"
                  cy="95"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="290"
                  cy="71"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="452"
                  cy="51"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="552"
                  cy="31"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="591"
                  cy="133"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="648"
                  cy="106"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="721"
                  cy="18"
                  fill="#ffffff"
                  r="7"
                />
              </g>
              <g>
                <path
                  d="M -10 170 l 60 -80 l 40 50 l 30 -40 l 30 50 l 30 -30 l 30 60 l 40 -30 l 60 20 l 60 -50 l 100 30 l 100 -30 l 50 30 l 50 -50 l 50 50 l 20 -40 l 40 60 v 100 h -900 z"
                  fill="#6a5082"
                />
                <path
                  d="M -30 220 l 70 -80 l 50 50 l 40 -40 l 30 50 l 30 -30 l 30 60 l 40 -30 l 60 20 l 60 -50 l 100 30 l 90 -30 l 50 30 l 40 -50 l 30 50 l 30 -40 l 40 60 l 30 -40 v 150 h -900 z"
                  fill="#5e3684"
                />
                <rect
                  fill="#183a23"
                  height="220"
                  width="770"
                  x="-10"
                  y="280"
                />
                <rect
                  fill="url(#tree)"
                  height="60"
                  width="750"
                  x="0"
                  y="230"
                />
              </g>
              <g>
                <ellipse
                  cx="500"
                  cy="430"
                  fill="#1a4227"
                  rx="140"
                  ry="40"
                />
                <ellipse
                  cx="675"
                  cy="330"
                  fill="#1a4227"
                  rx="145"
                  ry="30"
                />
                <ellipse
                  cx="40"
                  cy="370"
                  fill="#1a4227"
                  rx="150"
                  ry="45"
                />
                <ellipse
                  cx="375"
                  cy="340"
                  fill="#0f2917"
                  rx="190"
                  ry="40"
                />
                <rect
                  fill="brown"
                  height="250"
                  width="100"
                  x="320"
                  y="100"
                />
                <ellipse
                  cx="415"
                  cy="235"
                  fill="brown"
                  rx="12"
                  ry="25"
                  stroke="#a22121"
                  stroke-width="4"
                />
                <path
                  d="M 125 215 l 250 -230 l 250 230 z"
                  fill="#183a23"
                />
                <path
                  d="M 175 115 l 200 -230 l 200 230 z"
                  fill="#183a23"
                />
                <ellipse
                  cx="320"
                  cy="283"
                  fill="#b05000"
                  rx="30"
                  ry="35"
                />
                <rect
                  fill="#3d5279"
                  height="50"
                  width="32"
                  x="305"
                  y="300"
                />
                <ellipse
                  cx="320"
                  cy="315"
                  fill="#3d5279"
                  rx="33"
                  ry="20"
                />
                <ellipse
                  cx="320"
                  cy="346"
                  fill="#464bad"
                  rx="16"
                  ry="7"
                />
                <ellipse
                  cx="310"
                  cy="345"
                  fill="#555aaf"
                  rx="8"
                  ry="25"
                />
                <ellipse
                  cx="330"
                  cy="345"
                  fill="#555aaf"
                  rx="8"
                  ry="25"
                />
                <circle
                  cx="310"
                  cy="325"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="329"
                  cy="325"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="310"
                  cy="362"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="329"
                  cy="362"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="320"
                  cy="280"
                  fill="#ffaa56"
                  r="25"
                />
                <path
                  d="M 292,290 C 285,240  354,240  348.7,290 z"
                  fill="#bf5f00"
                />
              </g>
              <g>
                <rect
                  fill="url(#grass)"
                  height="40"
                  width="60"
                  x="100"
                  y="320"
                />
                <rect
                  fill="url(#grass)"
                  height="40"
                  width="60"
                  x="620"
                  y="280"
                />
                <rect
                  fill="url(#grass)"
                  height="40"
                  width="60"
                  x="550"
                  y="380"
                />
                <path
                  d="m 430,350 0,0 0,0 c -15,-120 85, -110 75,0 z"
                  fill="#6f6f6f"
                />
                <path
                  d="m 425,350 0,0 0,0 c -15,-110 85, -110 75,0 z"
                  fill="#9f9f9f"
                />
              </g>
            </svg>
          </div>
          <div>
            <p
              class="titlefont"
            >
              In the Black Forest
            </p>
          </div>
        </div>
        <div
          class="div3 inactive"
        >
          <canvas
            height="450"
            width="750"
          />
          <div>
            <p
              class="titlefont"
            >
              The Birch-Tree at Loschwitz
            </p>
          </div>
        </div>
        <div
          class="div4"
        >
          <div>
            <h1
              class="title"
            >
              A Wall Flower
            </h1>
          </div>
        </div>
        <div
          class="div5"
        >
          <canvas
            height="450"
            width="750"
          />
          <div
            class="btnWrapper"
          >
            <a
              class="button"
            >
              &gt;&gt;
            </a>
          </div>
        </div>
      </div>
      <div
        class="bigPoetry"
      >
        <div
          class="poetryBox"
        >
          <div />
        </div>
      </div>
    </div>
    <div
      class="sectionsplit"
    />
    <div
      class="sectionsplit"
    />
    <div
      class="tabBox"
    >
      <a
        href="#"
      >
        Tab 1
      </a>
      <a
        href="#"
      >
        Tab 2
      </a>
    </div>
    <div
      class="artWrapper"
    >
      <div
        class="art1"
      >
        <div>
          <div
            class="canvas"
          >
            <svg
              viewBox="0 0 750 450"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs>
                <pattern
                  height="40"
                  id="grass"
                  patternUnits="objectBoundingBox"
                  width="60"
                  x="0"
                  y="0"
                >
                  <path
                    d="m 25 38 l -20 -20 l 15 5 z"
                    fill="cadetblue"
                  />
                  <path
                    d="m 25 38 l 35 -30 l -25 10 z"
                    fill="#608da0"
                  />
                </pattern>
                <pattern
                  height="60"
                  id="tree"
                  patternUnits="userSpaceOnUse"
                  width="55"
                  x="-10"
                  y="-10"
                >
                  <rect
                    fill="black"
                    height="30"
                    width="13"
                    x="20"
                    y="30"
                  />
                  <path
                    d="m 0 52 l 27 -35 l 30 35 z"
                    fill="#black"
                  />
                  <path
                    d="m 2 42 l 25 -35 l 26 35 z"
                    fill="#black"
                  />
                  <path
                    d="m 4 32 l 23 -30 l 24 30 z"
                    fill="#black"
                  />
                </pattern>
                <pattern
                  height="60"
                  id="cross"
                  patternUnits="objectBoundingBox"
                  width="40"
                  x="0"
                  y="0"
                >
                  <rect
                    fill="white"
                    height="50"
                    width="40"
                    x="0"
                    y="0"
                  />
                </pattern>
              </defs>
              <g>
                <rect
                  fill="#0f0f18"
                  height="450"
                  width="750"
                  x="0"
                  y="0"
                />
                <circle
                  cx="34"
                  cy="46"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="131"
                  cy="53"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="105"
                  cy="95"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="290"
                  cy="71"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="452"
                  cy="51"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="552"
                  cy="31"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="591"
                  cy="133"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="648"
                  cy="106"
                  fill="#ffffff"
                  r="7"
                />
                <circle
                  cx="721"
                  cy="18"
                  fill="#ffffff"
                  r="7"
                />
              </g>
              <g>
                <path
                  d="M -10 170 l 60 -80 l 40 50 l 30 -40 l 30 50 l 30 -30 l 30 60 l 40 -30 l 60 20 l 60 -50 l 100 30 l 100 -30 l 50 30 l 50 -50 l 50 50 l 20 -40 l 40 60 v 100 h -900 z"
                  fill="#6a5082"
                />
                <path
                  d="M -30 220 l 70 -80 l 50 50 l 40 -40 l 30 50 l 30 -30 l 30 60 l 40 -30 l 60 20 l 60 -50 l 100 30 l 90 -30 l 50 30 l 40 -50 l 30 50 l 30 -40 l 40 60 l 30 -40 v 150 h -900 z"
                  fill="#5e3684"
                />
                <rect
                  fill="#183a23"
                  height="220"
                  width="770"
                  x="-10"
                  y="280"
                />
                <rect
                  fill="url(#tree)"
                  height="60"
                  width="750"
                  x="0"
                  y="230"
                />
              </g>
              <g>
                <ellipse
                  cx="500"
                  cy="430"
                  fill="#1a4227"
                  rx="140"
                  ry="40"
                />
                <ellipse
                  cx="675"
                  cy="330"
                  fill="#1a4227"
                  rx="145"
                  ry="30"
                />
                <ellipse
                  cx="40"
                  cy="370"
                  fill="#1a4227"
                  rx="150"
                  ry="45"
                />
                <ellipse
                  cx="375"
                  cy="340"
                  fill="#0f2917"
                  rx="190"
                  ry="40"
                />
                <rect
                  fill="brown"
                  height="250"
                  width="100"
                  x="320"
                  y="100"
                />
                <ellipse
                  cx="415"
                  cy="235"
                  fill="brown"
                  rx="12"
                  ry="25"
                  stroke="#a22121"
                  stroke-width="4"
                />
                <path
                  d="M 125 215 l 250 -230 l 250 230 z"
                  fill="#183a23"
                />
                <path
                  d="M 175 115 l 200 -230 l 200 230 z"
                  fill="#183a23"
                />
                <ellipse
                  cx="320"
                  cy="283"
                  fill="#b05000"
                  rx="30"
                  ry="35"
                />
                <rect
                  fill="#3d5279"
                  height="50"
                  width="32"
                  x="305"
                  y="300"
                />
                <ellipse
                  cx="320"
                  cy="315"
                  fill="#3d5279"
                  rx="33"
                  ry="20"
                />
                <ellipse
                  cx="320"
                  cy="346"
                  fill="#464bad"
                  rx="16"
                  ry="7"
                />
                <ellipse
                  cx="310"
                  cy="345"
                  fill="#555aaf"
                  rx="8"
                  ry="25"
                />
                <ellipse
                  cx="330"
                  cy="345"
                  fill="#555aaf"
                  rx="8"
                  ry="25"
                />
                <circle
                  cx="310"
                  cy="325"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="329"
                  cy="325"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="310"
                  cy="362"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="329"
                  cy="362"
                  fill="#143767"
                  r="8"
                />
                <circle
                  cx="320"
                  cy="280"
                  fill="#ffaa56"
                  r="25"
                />
                <path
                  d="M 292,290 C 285,240  354,240  348.7,290 z"
                  fill="#bf5f00"
                />
              </g>
              <g>
                <rect
                  fill="url(#grass)"
                  height="40"
                  width="60"
                  x="100"
                  y="320"
                />
                <rect
                  fill="url(#grass)"
                  height="40"
                  width="60"
                  x="620"
                  y="280"
                />
                <rect
                  fill="url(#grass)"
                  height="40"
                  width="60"
                  x="550"
                  y="380"
                />
                <path
                  d="m 430,350 0,0 0,0 c -15,-120 85, -110 75,0 z"
                  fill="#6f6f6f"
                />
                <path
                  d="m 425,350 0,0 0,0 c -15,-110 85, -110 75,0 z"
                  fill="#9f9f9f"
                />
              </g>
            </svg>
          </div>
        </div>
        <div>
          <div
            class="poetryBox"
          >
            <h2
              class="titlefont"
            >
              In The Black Forest
            </h2>
            <div />
          </div>
        </div>
      </div>
      <div
        class="art2"
      >
        <div>
          <canvas
            height="450"
            width="750"
          />
        </div>
        <div>
          <div
            class="poetryBox reversed"
          >
            <h2
              class="titlefont"
            >
              The Birch-Tree at Loschwitz
            </h2>
            <div />
          </div>
        </div>
      </div>
      <div
        class="art3"
      >
        <div>
          <canvas
            height="450"
            width="750"
          />
        </div>
        <div>
          <div
            class="poetryBox"
          >
            <h2
              class="titlefont"
            >
              A Wall Flower
            </h2>
            <div />
          </div>
        </div>
      </div>
    </div>
    <div
      class="sectionsplit"
    />
    <div
      id="root"
    />
  </main>
</div>
`;
